### DISCUSSION TITLE
Docs: Clearify if npm packages can be used

### DISCUSSION THREAD

**Discussion by MentalGear** [framesurge/perseus#333] [Q&A]
This looks really sweet, but most real world apps need to use npm packages. Can these be used with typescript support?

**Comment by arctic-hen7** [comment]
Perseus is built on top of Sycamore and `wasm-bindgen`, so absolutely! Honestly, you can usually find what you need in the Rust ecosystem, and most crates run on Wasm these days, so personally I've very rarely needed npm packages. That said, you can absolutely do JS interop through `wasm-bindgen`, see their docs for details.
